.github/workflows/re-build-boards.yml

```diff
@@ -1,11 +1,8 @@
-name: Build boards
+name: Reusable | Build boards
```

Comment:
Are there non-reusable versions of this? I'm not sure why you want the distinction here.

Reply:
No, this is just a normal workflow. The `workflow_call` event is what makes this reusable.
The reusable prefix in name is added just to separate it from the rest in GitHub UI.